doc/ACCOUNT_DERIVATION.md

```diff
@@ -13,11 +13,10 @@ At Sig.Network we use the [CAIP-2](https://chainagnostic.org/CAIPs/caip-2) stand
 
 The following examples show how different blockchain networks are identified using CAIP-2:
 - Solana Mainnet: `solana:5eykt4UsFv8P8NJdTREpY1vzqKqZKvdp`
-- Solana Devnet: `solana:EtWTRABZaYq6iMfeYKouRu166VU2xqa1`
 - Ethereum Mainnet: `eip155:1`
-- Ethereum Sepolia: `eip155:11155111`
 - Bitcoin Mainnet: `bip122:000000000019d6689c085ae165831e93`
-- Bitcoin Testnet: `bip122:000000000933ea01ad0ee984209779ba`
+
+For simplicity, we reuse Mainnet derivation paths for all dev and testnet networks. For example, Ethereum Sepolia will have the same derivation path as Ethereum Mainnet.
```

Comment:
Adding an Env variable appeared to be troublesome for both MPC, Tooling, as as a result - users. Also, it is kind of a duplication, since we already define our environment by the root MPC key.